packages/fxa-payments-server/src/routes/Subscriptions/index.test.tsx

```diff
@@ -810,16 +857,20 @@ describe('routes/Subscriptions', () => {
             webSubscription,
           ],
         },
+        mockPlans: MOCK_PLANS,
+        mockSubsequentInvoices: MOCK_SUBSEQUENT_INVOICES,
       });
 
-      const { findByTestId, queryAllByTestId } = render(<Subject />);
+      const blag = render(<Subject />);
```

Comment:
Is this necessary or was it just for testing?

Reply:
yes i  was just using it for a console.log, removing it now